gatsby-ssr.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { ThemeProvider } from "@emotion/react";
+import {darkTheme} from "~components/dev-hub/theme";
+import SurveyBanner from "~components/dev-hub/survey-banner";
+
+export const wrapPageElement = ({ element }) => {
+    return (
+
+        <ThemeProvider theme={darkTheme}>
+            {element}
+            <SurveyBanner></SurveyBanner>
+        </ThemeProvider>
+
+    );
+};
```

Comment:
```suggestion
export const wrapPageElement = ({ element }) => (
        <ThemeProvider theme={darkTheme}>
            {element}
            <SurveyBanner></SurveyBanner>
        </ThemeProvider>
    );
```

This is again a nitpick that doesn't block but we can omit the `{}` since we return JSX straight away.